docs/deployment/breaking-changes.md

```diff
@@ -0,0 +1,38 @@
+# Breaking changes and migrations
+
+This section lists breaking changes introduced in OpenCTI, per version starting with the latest.
+
+Please follow the migration guides if you need to upgrade your platform. 
+
+## OpenCTI 6.2
+
+Change to the observable "promote" return value: we now return the created Indicator instead of the original Observable.
```

Comment:
```suggestion
Change to the observable "promote" return value: the API now return the created Indicator instead of the original Observable.
```
```suggestion
Change to the observable "promote" return value: we now return the created Indicator instead of the original Observable.
```